fhireval/data/cmg-specimen.json

```diff
@@ -0,0 +1,208 @@
+{
+  "groupname": "CMG-Examples",
+  "resources": {
+    "Specimen" : [
+      {
+        "resourceType": "Specimen",
+        "identifier": [ {
+          "system": "urn:ncpi:unique-string",
+          "value": "ct.subject09"
+        }, {
+          "system": "urn:ncpi:unique-string",
+          "value": "FHIR-CMG-TEST|ct.subject09"
+        }, {
+          "system": "https://dbgap-api.ncbi.nlm.nih.gov/specimen",
+          "value": "1155303"
+        } ],
+        "subject": {
+          "reference": "Patient/4729"
+        }
+      }, {
+        "resourceType": "Specimen",
+        "identifier": [ {
+          "system": "urn:ncpi:unique-string",
+          "value": "ct.subject08"
+        }, {
+          "system": "urn:ncpi:unique-string",
+          "value": "FHIR-CMG-TEST|ct.subject08"
+        }, {
+          "system": "https://dbgap-api.ncbi.nlm.nih.gov/specimen",
+          "value": "1155302"
+        } ],
+        "subject": {
+          "reference": "Patient/4728"
+        }
+      }, {
+        "resourceType": "Specimen",
+        "identifier": [ {
+          "system": "urn:ncpi:unique-string",
+          "value": "ct.subject07"
+        }, {
+          "system": "urn:ncpi:unique-string",
+          "value": "FHIR-CMG-TEST|ct.subject07"
+        }, {              
+          "system": "https://dbgap-api.ncbi.nlm.nih.gov/specimen",
+          "value": "1155301"
+        } ],
+        "subject": {
+          "reference": "Patient/4727"
+        }
+      }, {
+        "resourceType": "Specimen",
+        "identifier": [ {
+          "system": "urn:ncpi:unique-string",
+          "value": "ct.subject06"
+        }, {
+          "system": "urn:ncpi:unique-string",
+          "value": "FHIR-CMG-TEST|ct.subject06"
+        }],       
+        "subject": {
+          "reference": "Patient/4726"
+        },
+        "collection": {
+          "bodySite": {
+            "coding": [ {
+              "system": "https://uberon.github.io/",
```

Comment:
`"http://purl.obolibrary.org/obo/uberon.owl"`

Reply:
Good catch. This has been fixed.